browser/src/core/Socket.js

```diff
@@ -953,6 +954,7 @@ app.definitions.Socket = L.Class.extend({
 							if (value === 'discard') {
 								// They want to refresh the page and load document again for all
 								that.sendMessage('closedocument');
+								window.location.reload();
```

Comment:
This should be fine, although I think it worked previously simply because we always reloaded when we got disconnected, and the 'closedocument' will disconnect us from WSD. At any rate, the `reload()` call should be fine, I expect.

Reply:
Thanks for checking :)